skimage/exposure/exposure.py

```diff
@@ -662,9 +663,9 @@ def adjust_gamma(image, gamma=1, gain=1):
 
     Examples
     --------
-    >>> from skimage import data, exposure, img_as_float
-    >>> image = img_as_float(data.moon())
-    >>> gamma_corrected = exposure.adjust_gamma(image, 2)
+    >>> import skimage as ski
+    >>> image = ski.img_as_float(ski.data.moon())
```

Comment:
```suggestion
    >>> image = ski.util.img_as_float(ski.data.moon())
```
"utility functions like `img_as_float` have to be accessed via the `util` subpackage" from the [skimage2 roadmap](https://github.com/scikit-image/scikit-image/wiki/API-changes-for-skimage2)